.eslintrc.js

```diff
@@ -11,16 +12,120 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaFeatures: {
-      jsx: true
+      jsx: true,
     },
     ecmaVersion: 'latest',
-    sourceType: 'module'
+    sourceType: 'module',
   },
-  plugins: ['react', '@typescript-eslint'],
+  plugins: ['react', '@typescript-eslint', 'spellcheck'],
   rules: {
     semi: ['error', 'always'],
     'space-before-function-paren': ['error', 'never'],
     'comma-dangle': ['error', 'only-multiline'],
-    'object-property-newline': ['error', { allowAllPropertiesOnSameLine: false }],
-  }
+    'object-property-newline': [
+      'error',
+      { allowAllPropertiesOnSameLine: false },
+    ],
+    'spellcheck/spell-checker': [
+      1,
+      {
+        comments: true,
+        strings: true,
+        identifiers: true,
+        templates: true,
+        lang: 'en_US',
+        skipWords: [
+          'dict',
+          'aff',
+          'hunspellchecker',
+          'hunspell',
+          'utils',
+          'Enum',
+          'Reddit',
+          'href',
+          'Mvvm',
+          'Actuals',
+          'Lyyyy',
+          'Mthly',
+          'Qtly',
+          'Vestings',
+          'ftes',
+          'curr',
+          'teammember',
+          'pathname',
+          'evenodd',
+          'Checkbox',
+          'favicon',
+          'webp',
+          'jpeg',
+          'Debounce',
+          'calc',
+          'callout',
+          'youtube',
+          'bgcolor',
+          'Colum',
+          'mips',
+          'Youtube',
+          'scrollbar',
+          'webkit',
+          'nowrap',
+          'Gmail',
+          'fulltime',
+          'Mockstore',
+          'Renderer',
+          'Rect',
+          'centric',
+          'project',
+          'industry',
+          'scalable',
+          'Protocol’s',
+          'sustainably',
+          'overscroll',
+          'Makerdao',
+          'goto',
+          'listitem',
+          'industry’s',
+          'project’s',
+          'thead',
+          'labelledby',
+          'Roboto',
+          'Cantarell',
+          'makerburn',
+          'spacings',
+          'visibilitychange',
+          'Hight',
+          'Subheader',
+          'Identicon',
+          'ident',
+          'Unordered',
+          'juan',
+          'Julien',
+          'wouter',
+          'Kampmann',
+          'Wouter',
+          'Tooltip',
+          'Team’s',
+          'Petru',
+          'Catana',
+          'Jevans',
+          'onchain',
+          'Whitepaper',
+          'Onboarding',
+          'gtag',
+          'mailto',
+          'Millis',
+          'Datetime',
+          'Sitemap',
+          'Mip', // Explicity word to talk about mips
+        ],
```

Comment:
Can we move this array to a different file .json to make the config more readable?

Reply:
you are right, I will move to array